src/mappings/v2/equippable.ts

```diff
@@ -0,0 +1,38 @@
+import { burned, plsBe, plsNotBe, real } from '@kodadot1/metasquid/consolidator'
+import { getOrFail as get } from '@kodadot1/metasquid/entity'
+import { Optional } from '@kodadot1/metasquid/types'
+import { Equippable, Interaction } from '@kodadot1/minimark/v2'
+
+import { NFTEntity } from '../../model'
+import { createEvent } from '../shared/event'
+import { unwrap } from '../utils'
+import { isOwnerOrElseError } from '../utils/consolidator'
+import { error, success } from '../utils/logger'
+import { Action, Context } from '../utils/types'
+import { getAs } from './getters'
+
+const OPERATION = Action.EQUIPPABLE
+
+export async function equippable(context: Context) {
+  let interaction: Optional<Equippable> = null
+
+  try {
+    const getE = getAs<Interaction.EQUIPPABLE>()
+    const { value: equip, caller, timestamp, blockNumber, version } = unwrap(context, getE);
+    interaction = equip
+    const nft = await get<NFTEntity>(context.store, NFTEntity, interaction.id)
+    plsBe(real, nft)
+    plsNotBe(burned, nft)
+    isOwnerOrElseError(nft, caller)
+    nft.updatedAt = timestamp
+
+    // TODO: add logic for EQUIPing resource
```

Comment:
Will this be done in this PR?

Reply:
most probably not I'll plan to check critical issues in this comment but most probably merge as is

Reply:
https://github.com/kodadot/rubick/issues/217